incubator/zalenium/README.md

```diff
@@ -0,0 +1,125 @@
+# ZALENIUM-HELM-CHART
+======================================================
+
+## Introduction
+
+This chart bootstraps a zalenium deployment on a kubernetes cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- teracy-dev v0.6.0-a4+ (https://github.com/teracyhq/dev)
+- Follow this doc (https://github.com/teracyhq-incubator/teracy-dev-entry-k8s) to set up teracy-dev.
+- helm v2.10.0+. Default installed in teracy-dev.
```

Comment:
We should have it like this:
# Prerequisite
- `vagrant` (link to vagrant download docs)
- `virtualbox` (link to virtualbox download docs)
# How to get it works
1. Setup the VM and local Kubectl Cluster
```
$ cd ~/
$ git clone https://github.com/teracyhq/dev.git k8s-dev
$ cd k8s-dev/
$ TERACY_DEV_ENTRY_LOCATION_GIT=https://github.com/teracyhq-incubator/teracy-dev-entry-k8s.git \
  TERACY_DEV_ENTRY_LOCATION_BRANCH=develop TERACY_DEV_ENTRY_LOCATION_SYNC=true \
  vagrant up
```
  - Explain why we need `teracy-dev` (example: For it to work on you machine with consistent workflow, ...), then ref link to teracy-dev
  - Then you will see the`workspace/teracy-dev-entry` has included the settings required for this helm-chart to work, lets visit https://github.com/teracyhq-incubator/teracy-dev-entry-k8s for detail config...
  - Your setup has install `teracy-dev-k8s` to act as your local Kubectl Cluster, (possible more intro for `teracy-dev-k8s` here)

2. Setup `zalenium` from the scratch
  - The instruction the install this helm-chart here
  - Make sure to have it as details as possible (example: delete old one before start a new one, add a sample config)
3. Preview, test the `zelanium` helm-chart
  - The instruction to preview (example: goto k8s.local:4444)
```

Reply:
some parts should move to the teracy-dev-entry-k8s docs instead. It can be assumed that users follow the docs on teracy-dev-entry-k8s will have a working local k8s cluster to use.